data/coalition/lunarium intro.txt

```diff
@@ -1619,30 +1619,30 @@ mission "Lunarium: Combat Training 1"
+			`	"There you are, Captain <last>," she greets you. "Pyakri, my name is, and looking for your transport services, my friends and I are. Talk in your ship, could we? Wish to discuss the details there, I do."`
 			choice
 				`	"Sure, let me get the hatch."`
 				`	"You'll have to clear out, I'm not taking any passengers now."`
 					decline
-			`	You bring her aboard, and she continues when you close the hatch. "Thank you Captain. Now, assisted some of my colleagues, you already have, yes?" She waits for you to answer, but seeing your puzzled look, resumes her explanation. "The Lunarium Redoubt? Assisted us before, you have, even if aware of all the details beforehand you were not. Suppose I do that mostly hear of us as criminals or the like, you must. Petty in their propaganda, the Heliarchs are. Recommended you as trustworthy, some you've helped have, so that possible to get on with a special endeavor I've been planning, it is."`
+			`	You bring her aboard, and she continues when you close the hatch. "Thank you Captain. Now, assisted some of my colleagues, you already have, yes?" She waits for you to answer, but seeing your puzzled look, resumes her explanation. "The Lunarium Redoubt? Assisted us before, you have, even if aware of all the details beforehand you were not. Suppose I do that mostly heard of us as criminals or the like, you must have. Petty in their propaganda, the Heliarchs are. Recommend you as trustworthy, some people did whom helped you have, so possible it is, to go ahead with a special endeavor that planning, I have been."`
 			choice
 				`	"What's this 'endeavor' you need help with?"`
 					goto help
 				`	"Your friends only told me what I was really doing after I was done. I don't want to help you any further."`
-			`	"Why, interested are you in helping the Heliarchs instead? Worked with them, some of our members had. An arbiter, Elirom, one of them, was. One of the highest ranks, that is, second only to the consuls themselves," she responds, getting closer to you. "Change, for the better, the Coalition soon will. Difficult it will be, to take down the Heliarchy. Very strong they are, so greatly appreciated any help would be, and appropriately reward those who helped us, we will, once free from them we are."`
+			`	"Why, interested are you in helping the Heliarchs instead? Worked with them, some of our members have. An arbiter, Elirom, one of them, was. One of the highest ranks, that is, second only to the consuls themselves," she responds, getting closer to you. "Change for the better, the Coalition soon will. Difficult it will be, to take down the Heliarchy. Very strong it is, so greatly appreciated any help would be. Of course, once free from them we are, appropriately reward, we will, help us, those who did."`
```

Comment:
That end reads very odd, with the many commas pausing over and over.

Reply:
Maybe `... shall be those, who in our time of need do help us."` ? Or `..., who do help us in our time of need."` ?